_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
+author: "Tuli Nivas, Geeta Shankar, and Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Salesforce.png
+show_pagenav: false
+---
+
+Rolling into our next presentation of [Trino Fest 2023]({% post_url
+2023-06-20-trino-fest-2023-recap %}), we're excited to bring you
+Tuli Navas and Geeta Shankar's talk from the Performance Engineering Team at
+Salesforce. They provide numerous reasons for why they need Trino and
+further explain how it is essential for anomaly detection within
+their data. It's an insightful talk about using a query engine to ensure data
+quality and how switching to Trino has massively improved their performance. You definitely
```

Comment:
You should set up a character length limit and auto-wrapping in your IDE. We keep lines to a max length of 80 characters, and every line should be as close to 80 characters without going over.

This applies to several other places as well, so please make sure you get it fixed.